cs/test/expressions/Program.cs

```diff
@@ -108,6 +108,29 @@ public SerializerDebugView(Type type)
         string IDebugView.DebugView { get { return debugView; } }
     }
 
+    public static class BondTypeAliasConverter
+    {
+        public static int ConvertToDecimalCount = 0;
```

Comment:
do we need this counters on test/expressions/Program? Don't see it being used.

Reply:
Removed.